### ISSUE TITLE
[Feature]: ability to make archive

### ISSUE THREAD

**Issue by NSTAdventure** [Raival-e/File-Explorer#58]
### Feature Description

Allow file explorer app to create archive with following popular or used format:
- RAR, ZIP
- 7z

Additionally you can set compression level from no compression to the best compression and allow user to enable solid archive + split into multiple volume

### Duplicate feature

- [X] This feature has not been requested yet.

**Comment by Raival-e** [comment]
You can make archives by selecting the files/folders you want to archive, then long click on one of the files selected, and choose compress.

Though yes it can be improved. Thanks for the suggestion.

**Comment by NSTAdventure** [comment]
> You can make archives by selecting the files/folders you want to archive, then long click on one of the files selected, and choose compress.

I see but it would be very cool if you improve it like winrar by rarlab app or others

> Though yes it can be improved. Thanks for the suggestion.

Yes yw it's suprising this app development are still alive thought

**Comment by OkyDooky** [comment]
Yeah, any additional features/options for both compression and decompression would be really appreciated, since it seems there is a deficiency (at least with open source) of good archiving apps for Android.
Maybe open source desktop apps like Ark or PeaZip could be good references for implementing more advanced features and handling more archive formats. It would be nice to make archives other than just .zips and specify level of compression or other settings.
